measurements/toxicity/toxicity.py

```diff
@@ -0,0 +1,146 @@
+    `max_toxicity`: the maximum toxicity over all scores (if `aggregation` = `maximum`)
+    `toxicity_ratio`": the percentage of predictions with toxicity >= 0.5 (if `aggregation` = `ratio`)
+
+Examples:
+
+    Example 1 (default behavior):
+        >>> toxicity = evaluate.load("toxicity", module_type="measurement")
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts)
+        >>> print(results)
+        {'toxicity': [0.0002070277841994539, 0.856372594833374]}
+
+    Example 2 (returns ratio of toxic sentences):
+        >>> toxicity = evaluate.load("toxicity", module_type="measurement")
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, aggregation = "ratio")
+        >>> print(results)
+        {'toxicity_ratio': 0.5}
+
+    Example 3 (returns the maximum toxicity score):
+
+        >>> toxicity = evaluate.load("toxicity", module_type="measurement")
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, aggregation = "maximum")
+        >>> print(results)
+        {'max_toxicity': 0.856372594833374}
+
+    Example 4 (uses a custom model):
+
+        >>> toxicity = evaluate.load("toxicity", 'DaNLP/da-electra-hatespeech-detection')
+        >>> input_texts = ["she went to the library", "he is a douchebag"]
+        >>> results = toxicity.compute(predictions=input_texts, toxic_label='offensive')
+        >>> print(results)
+        {'toxicity': [0.017554517835378647, 0.020320769399404526]}
+"""
+
+
+def toxicity(preds, toxic_classifier, toxic_label):
+    toxic_scores = []
+    if toxic_label in toxic_classifier.model.config.id2label.values():
+        for pred in preds:
+            pred_toxic = toxic_classifier(str(pred))
+            hate_toxic = [r["score"] for r in pred_toxic if r["label"] == toxic_label][0]
+            toxic_scores.append(hate_toxic)
+    else:
+        logger.warning(
+            "The toxic label that you specified is not part of the model labels. Run `model.config.id2label` to see what labels your model outputs."
+        )
+    return toxic_scores
+
+
+@evaluate.utils.file_utils.add_start_docstrings(_DESCRIPTION, _KWARGS_DESCRIPTION)
+class Toxicity(evaluate.Measurement):
+    def _info(self):
+        return evaluate.MeasurementInfo(
+            module_type="measurement",
+            description=_DESCRIPTION,
+            citation=_CITATION,
+            inputs_description=_KWARGS_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "predictions": datasets.Value("string", id="sequence"),
+                }
+            ),
+            codebase_urls=[],
+            reference_urls=[],
+        )
+
+    def _download_and_prepare(self, dl_manager):
+        if self.config_name == "default":
```

Comment:
Again, this avoids some indents and duplicate code:
```suggestion
        if self.config_name == "default":
            model_name = "facebook/roberta-hate-speech-dynabench-r4-target"
        else:
            model_name = self.config_name
```
You need to change the naming below, too.